lib/utf8utils.c

```diff
@@ -33,7 +33,7 @@ _is_character_unsafe(gunichar uchar, const gchar *unsafe_chars)
   if (!unsafe_chars)
     return FALSE;
 
-  return strchr(unsafe_chars, (gchar) uchar) != NULL;
+  return _strchr_optimized_for_single_char_haystack(unsafe_chars, (gchar) uchar) != NULL;
```

Comment:
If the _strchr_optimized_for_single_char_haystack is used only from utff8utils.c, why is it in different header file.
To be honest I don't really think, that this function will be used from another source